src/services/item/plugins/importExport/service.ts

```diff
@@ -207,6 +206,68 @@ export class ImportExportService {
     }
   }
 
+  async fetchItemData(
+    actor,
+    repositories,
+    item,
+  ): Promise<{ name: string; stream?: NodeJS.ReadableStream; buffer?: Buffer; mimetype: string }> {
```

Comment:
We return either a stream of a buffer ? Would it be possible to return a stream in all cases ?